### PR TITLE
Add the py-dryscrape package

### PR DESCRIPTION
**Pull request by krafczyk**

### CODE REVIEW

File: var/spack/repos/builtin/packages/py-dryscrape/package.py [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+       for Python"""
+
+    homepage = "https://github.com/niklasb/dryscrape"
+    url      = "https://pypi.io/packages/source/d/dryscrape/dryscrape-1.0.tar.gz"
+
+    version('development', git="https://github.com/niklasb/dryscrape",
+            branch="master")
+    version('1.0', '267e380a8efaf9cd8fd94de1639d3198')
+
+    depends_on('py-setuptools', type='build')
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-dryscrape/package.py]: This package doesn't use setuptools.

File: var/spack/repos/builtin/packages/py-dryscrape/package.py [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    depends_on('py-lxml', type=('build', 'run'))
+    depends_on('py-webkit-server', type=('build', 'run'))
```
Comment (adamjstewart) [inline var/spack/repos/builtin/packages/py-dryscrape/package.py]:
```
py-webkit-server@1.0:
```

File: var/spack/repos/builtin/packages/py-dryscrape/package.py (hunk 1 above)
Comment (krafczyk) [inline var/spack/repos/builtin/packages/py-dryscrape/package.py]: Fixed.

File: var/spack/repos/builtin/packages/py-dryscrape/package.py (hunk 2 above)
Comment (krafczyk) [inline var/spack/repos/builtin/packages/py-dryscrape/package.py]: Fixed.

**Comment by krafczyk** [comment]
@adamjstewart  I also changed development to develop like you mentioned in #4092.

**Pull request merged**